frontend/src/pages/ActivityReport/index.js

```diff
@@ -293,16 +293,31 @@ function ActivityReport({
   const reportCreator = { name: user.name, role: user.role };
   const tagClass = formData.calculatedStatus === REPORT_STATUSES.APPROVED ? 'smart-hub--tag-approved' : '';
 
+  const author = formData.author ? (
+    <>
+      <hr />
+      <p>
+        <strong>Creator:</strong>
+        {' '}
+        {formData.author.name}
```

Comment:
I think we might want the role(s) here too. At least that's what we see in Jira.